extensions/EXT/SPV_EXT_optnone.html

```diff
@@ -0,0 +1,706 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="generator" content="Asciidoctor 2.0.16">
```

Comment:
Looks like this was not generated with the version of asciidoctor provided in the docker image. We're working towards automatically generating and publishing specs (see https://github.com/KhronosGroup/SPIRV-Registry/issues/270) and one of the necessary steps involves making sure that all specs can be generated with the version provided in the docker image. It seems that we have not updated EXT extensions yet and this spec looks rather straightforward so I think re-generating the HTML needn't be a blocker to this PR going in (but would be nice to do if you can).